src/GovUk.Education.ExploreEducationStatistics.Data.Processor.Tests/Services/ImporterLocationServiceTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Common.Services.Interfaces;
+using GovUk.Education.ExploreEducationStatistics.Data.Model;
+using GovUk.Education.ExploreEducationStatistics.Data.Processor.Services;
+using Moq;
+using Xunit;
+using static GovUk.Education.ExploreEducationStatistics.Data.Model.Database.StatisticsDbUtils;
+namespace GovUk.Education.ExploreEducationStatistics.Data.Processor.Tests.Services
+{
+    public class ImporterLocationServiceTests
+    {
+        [Fact]
+        public async Task Find_EnglishDevolvedAreaCaseSensitiveLocationName()
+        {
+            var country = new Country("1", "england");
+            var lowerCase = new EnglishDevolvedArea("2", "casesensitive");
```

Comment:
Both of these need to have the same code otherwise you aren't testing case sensitivity at all since they are two different locations by their code and name combination regardless of case. At the moment, one has code 1, the other code 2.